### ISSUE TITLE
plot_AbanicoPlot() crashes with plot = FALSE

### ISSUE THREAD

**Issue by mcol** [R-Lum/Luminescence#221]
Strangely, the function complains about plot limits when asked not to plot:
```R
data(ExampleData.DeValues, envir = environment())
plot_AbanicoPlot(ExampleData.DeValues$CA1, plot = TRUE)  # this is OK

plot_AbanicoPlot(ExampleData.DeValues$CA1, plot = FALSE) # this crashes
# Error in plot.window(...) : need finite 'xlim' values
```

**Comment by mcol** [comment]
It turns out that `plot_AbanicoPlot()` doesn't take the `plot` argument, so when I set `plot = FALSE` it interprets it as `plot.ratio = FALSE`, which is then cast to 0 when computing `xlim = c(limits.x[1], limits.x[2] * (1 / plot.ratio))`, thus generating an infinity.

I'll add some validation for `plot.ratio` to catch that early on.

**Comment by mcol** [comment]
Fixed by #222.
